bootstrap.sh

```diff
@@ -96,6 +97,7 @@ the location where LPF is installed"
 reference=yes
 hyperdags=yes
 hyperdags_using=reference
+nonblocking=yes
```

Comment:
The nonblocking flag is never checked for in the remainder of this script. I believe it should be appended to the if-statement in line 264.

Reply:
The script should also override `WITH_NONBLOCKING_BACKEND` to `OFF` when `nonblocking` equals `no` by passing the corresponding flag to `cmake` later on.